test/config/integration/certs/upstreamcert.cfg

```diff
@@ -36,3 +36,5 @@ authorityKeyIdentifier = keyid:always
 DNS.1 = *.lyft.com
 IP.1 = 0.0.0.0
 IP.2 = ::
```

Comment:
Do we still need these two address then?

Reply:
I was hedging my bets because I wasn't sure if the community would want to switch from AnyAddress to Loopback on all platforms or just for Windows.

I can go ahead and delete that if we stick with Loopback on all platforms

Reply:
these are only used in tests anyway. Isn't this PR changed all anyaddress to loopback?

Reply:
I changed in ab2223043a9f5b23e6876d7c497a3e91445b6da5, I will do some more iterations to see if more changes are needed in other tests as well